omero_marshal/decode/decoders/original_file.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2015 Glencoe Software, Inc. All rights reserved.
+#
+# This software is distributed under the terms described by the LICENCE file
+# you can find at the root of the distribution bundle.
+# If the file is missing please request a copy by contacting
+# jason@glencoesoftware.com.
+#
+
+from .. import Decoder
+from omero.model import OriginalFileI
+
+
+class OriginalFileDecoder(Decoder):
+
+    TYPE = 'TBD#OriginalFile'
+
+    OMERO_CLASS = OriginalFileI
+
+    def decode(self, data):
+        v = super(OriginalFileDecoder, self).decode(data)
+        self.set_property(v, 'path', data.get('path'))
+        self.set_property(v, 'size', data.get('size'))
+        self.set_property(v, 'atime', data.get('atime'))
+        self.set_property(v, 'mtime', data.get('mtime'))
+        self.set_property(v, 'ctime', data.get('ctime'))
+        self.set_property(v, 'hash', data.get('hash'))
+        self.set_property(v, 'mimetype', data.get('mimetype'))
+        self.set_property(v, 'name', data.get('name'))
```

Comment:
Understood. I guess I'm always seeing things from an OMERO point of view, forgetting that OME model is different.
It might be slightly more convenient to users to use "Name" (as on all other objects) instead of "name", but if it's useful for the lowercase attributes to distinguish OME vv OMERO differences then fair enough.
If we want to indicate that OME has no ```OriginalFile```, should the original file be under ```'omero:File'``` in the File Annotation encoding?